src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandlerDocPreviewTest.java

```diff
@@ -0,0 +1,266 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
```

Comment:
I would suggest adding these test to JudicialDecesionHandlerTest rather than creating new test class only for one functionality of Documents

Reply:
I will look into it. Thanks